### ISSUE TITLE
Second iteration of exports

### ISSUE THREAD

**Issue by DavidBruant** [MyWebIntelligence/MyWebIntelligence#75]
Second iteration of #14 
Decide page/site graph, decide if CSV and which columns.

**Comment by DavidBruant** [comment]
Maybe OPML ?

**Comment by DavidBruant** [comment]
Decision is 3 exports:
* Page CSV
* Page GEXF
* Domain GEXF

These are per-territoire

Properties for site exports
```
post_url
post_title
post_excerpt
som_motscles
post_date_publication
post_content
post_contentlength
domain_title
domain_type
```

Properties for domain exports
```
domain_title
domain_type
count_post
```

For GEXF domain exports, edges have a weight corresponding to how many links have been seen between the two domain for the given direction.

**Comment by DavidBruant** [comment]
Closed via #86 . Create new issues for followups.
